components/function-runtimes/java11/examples/hello-world/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.project.kyma.serverless</groupId>
+    <artifactId>java-function</artifactId>
+    <version>0.0.1</version>
+    <packaging>jar</packaging>
+
+    <name>hello-world</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+
+    <dependencies>
```

Comment:
Those deps are needed to work with function in IDE, those deps are available in runtime image as I am doing merge of project.
Should we default the `deps` to this `pom.xml` or leave it empty?